scripts/predict.py

```diff
@@ -1,45 +1,59 @@
+from __future__ import print_function
 import json
 import os
+import re
 import pickle
 import sys
+import click
 from os.path import dirname, abspath
 sys.path.append(dirname(dirname(abspath(__file__))))
 from pkspace.utils.loaders import PKSpaceLoader # noqa
-import click # noqa
 
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
-              help='Path where pretrained model')
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='Path where pretrained model is located')
 @click.option('--output', default=None,
-              help='Name of output file for predicted output')
-def load_and_predict(mask_path, picture_path, model_path, output):
-    for file in (mask_path, picture_path, model_path):
-        if not os.path.exists(file):
-            print(file, "Path does not exist")
-            return
+              help='Name of file for predicted output')
+@click.option('--img_suffix', default='.png',
+              help='Suffix of predicted image')
+def load_and_predict(mask_path, picture_path, model_path, output, img_suffix):
     if output is None:
-        output = "{}_out.json".format(os.path.splitext(picture_path)[-2])
+        output = '{}_out.json'.format(os.path.splitext(picture_path)[-2])
+    elif output.endswith(img_suffix):
+        output = re.sub(img_suffix + '$', '.json', output)
+    elif not output.endswith('.json'):
+        output = '{0}{1}'.format(output, '.json')
 
-    if os.path.splitext(mask_path)[-1] == ".json":
+    if not os.path.exists(dirname(output)):
+        os.makedirs(dirname(output))
+
+    if os.path.splitext(mask_path)[-1] == '.json':
         loader = PKSpaceLoader()
         pictures, _ = loader.load_pic(picture_path, mask_path)
```

Comment:
@Adman probably would also be nice to check if this is really a image. You can use `imghdr` library.